### ISSUE TITLE
Simple cache wrapper with timeout

### ISSUE THREAD

**Issue by bnomei** [k-next/plugins#86]
Cache any data until set expiration time.

https://github.com/bnomei/kirby3-lapse

a port of https://github.com/jenstornell/kirby-time-cache with minor additions.

**Comment by jenstornell** [comment]
@bnomei At first sight it looks a bit more complicated than mine, but I have not tried it out so I don't really know.

Or, wait... Is that 3 examples in that code example? To make it looks less complicated, maybe just split them up into 3 examples?

If your plugin is good enough I'll close mine after Kirby 3 has been released.

**Comment by bnomei** [comment]
@jenstornell i did split the examples like you suggested.

**Comment by jenstornell** [comment]
https://github.com/k-next/getkirby.com/issues/13#event-1822268630

I read about the new Kirby cache. It seems like it will be built into the core.

**Comment by bnomei** [comment]
yeah. i use that internally. its just a one-function-wrapper around regular cache plus closure support.
